lib/MusicBrainz/Server/Report/ReleasesWithoutVACredit.pm

```diff
@@ -0,0 +1,45 @@
+package MusicBrainz::Server::Report::ReleasesWithoutVACredit;
+use Moose;
+use MusicBrainz::Server::Constants qw( $VARTIST_ID );
+
+with 'MusicBrainz::Server::Report::ReleaseReport';
+
+sub query {
+    "
+        SELECT
+            r.id AS release_id,
+            row_number() OVER (ORDER BY r.artist_credit, r.name)
+        FROM release r
+        JOIN artist_credit_name acn on acn.artist_credit = r.artist_credit
+        WHERE acn.artist = $VARTIST_ID AND acn.name != 'Various Artists'
```

Comment:
It was mentioned on IRC that the release reports should probably also handle tracks, i.e. find tracks that meet those conditions, get the release via track -> medium -> release, and `UNION` the two queries.